Brewfile

```diff
@@ -7,6 +7,6 @@ cask "docker"
 cask "ngrok"
 
 brew "nginx", restart_service: :changed
-brew "postgresql@9.4"
+brew "postgresql"
```

Comment:
Looks like we are using Postgres `9.4.21` in production. Might be best if we can avoid changing this 😕 @d12 what do you think?

Reply:
Or maybe it's time to bump the production version?

Reply:
Yeah I don't like unpinning the local version to be separate from the deployed version, and postgresql 9.4 will still install on each of our machines, but it seems the `pg` gem will not install with that version of psql properly on OSX Mojave, at least according to our experiences today... looking online to see if that's by design or we all just got unlucky in some way. 

In any event, postgresql 12 is about to release, so there's maybe that to consider too.

Reply:
Weird... Yeah if there's a way to make this work without unpinning the version, that'd be best. Let me know what you find.

Reply:
Hello! It turns out it wasn't a compatibility issue between operating systems, but a subtlety (read: annoyance) in how Homebrew now manages pinned packages: when you pin a version, as we are with postgresql, Homebrew installs it as "keg-only" meaning it is not linked anywhere systemwide (meaning it's not on PATH, etc) out of fear it could interfere/conflict with the latest installed version of that same application, and leaves that decision to the user.

I've changed the bootstrap script such that it will install and link postgresql@9.4 _only_ if there is no other version of postgresql detected. I can also put it in a y/n prompt if that's of interest. 

Now pg builds properly against version 9.4!

Reply:
Nice find :+1: This seems perfect for now, we can always iterate on the script in the future. 💖